README.md

```diff
@@ -1,2 +1,168 @@
+
+## /medications
+
+In this section, we describe information regarding all the medications to be specified in the Engage-HF context. These medications may be used by a clinician for medication requests to a patient (/users/$userId$/medicationRequests) or contra-indications (/users/$userId$/allergyIntolerances).
+
+### /medications/$medicationId$
+
+Based on [FHIR Medication](https://hl7.org/fhir/R4B/medication.html):
+
+|Property|Type|Values|Comments|
+|-|-|-|-|
+|id|string|-|[Resource](https://hl7.org/fhir/R4B/resource.html): Logical id of this artifact|
+|meta|Meta|-|[Resource](https://hl7.org/fhir/R4B/resource.html#Meta): Metadata about the resource|
+|identifier|list of Identifier|-|Business identifier for this medication|
+|extension|list of Extension|-|See /medications/$medicationId$/extension for possible values|
+
+Technically, we could use [FHIR MedicationKnowledge](https://hl7.org/fhir/R4B/medicationknowledge.html), but it doesn't seem to be ready for use - or at least take over the `medicineClassification` property from it.
+
+### /medications/$medicationId$/extension
+
+Based on the [Extension](https://hl7.org/fhir/R4B/extensibility.html#Extension) format specified in FHIR, a medication may contain a list of these following properties. Each property will need to get a url assigned to fit the FHIR data format.
+
+|Property|Type|Values|Comments|
+|-|-|-|-|
+|medicationClass|string|e.g. "BETABLOCKER"|A medicationClassId referring to a medicationClass specified in /medicationClasses/$medicationClassId$. One medication object may contain multiple medicationClass extension properties.|
+|minimumDailyDose|double|e.g. 6.25|Unit: mg/day. May only occur once.|
+|targetDailyDose|double|e.g. 50.0|Unit: mg/day. May only occur once.|
+
+Q: Do we possibly need a connection from a single medication to a video?
+
+### /medicationClasses/$medicationClassId$
+
+|Property|Type|Values|Comments|
+|-|-|-|-|
+|id|string|e.g. "BETABLOCKER"||
+|name|LocalizedText|-|A name for a given medicationClass to be displayed to a user.|
+|videoSectionId|string|-|To be used to retrieve a video using /videos/$videoSectionId$/videos/$videoId$.|
+|videoId|string|-|To be used to retrieve a video using /videos/$videoSectionId$/videos/$videoId$.|
+
+Q: Do we necessarily need a connection between medication classes and videos / videoSections?
+
+## /videoSections
+
+In this section, we describe all data related to educational videos to be shown in the Engage-HF mobile apps. The videos are grouped into different categories to be displayed as sections in the mobile apps.
+
+### /videoSections/$videoSectionId$
+
+|Property|Type|Values|Comments|
+|-|-|-|-|
+|title|LocalizedText|e.g. "ENGAGE-HF Application"|May be localized.|
+|description|LocalizedText|e.g. "Helpful videos on the ENGAGE-HF mobile application."|May be localized. Are there different videos / videoSections for each platform?|
+
```

Comment:
I could imagine that a thumbnail might also be good

Reply:
You are right - thanks! We might also want to check if there is an easy way to request a thumbnail from youtube directly, when we receive a youtube url on a client app. Depends on whether we want this kind of knowledge about possible urls in the client or be more independent

Reply:
Would be great if we can add this as a question for Monday. Embedding a Youtube video would give that to us for free, something else might require a dedicated thumbnail.

Reply:
YouTube-only is the way to go, I guess. If we decide to support another media source as well in a later update of the app(s), we can simply only show the title and link to the video directly or we hide the video in the old app version and only show it in the new one then.